test/.lando.yml

```diff
@@ -0,0 +1,31 @@
+testVar: testingtesting
```

Comment:
Sorry, I'm not sure I get what you mean, leave users to manually create this?

Reply:
Nope, you can dynamically write the yaml file at runtime, see https://github.com/lando/lando/blob/master/plugins/lando-app/test/functional/cli/app.spec.js#L42

Reply:
@dustinleblanc If that's an acceptable replication of the true functionality I'll just add it to the test setup.  👍

This test currently actually includes the fact it can read the yaml file in the first place. Have you included that in the tests you're creating? Would it be worth me adding that to my tests?

Reply:
Oh, you write it to file! Ignore me. 🤦🏻‍♂️

Reply:
ahaha no worries! Thanks for being so receptive to my knit picking! I think this pattern keeps the test fixture data right with the test itself rather than polluting the global test directories. If on the other hand we want to source the same test file for a bunch of tests, then I think your approach would have been the better way to go. Pumped to see this get merged when its all done and shiny!